.github/workflows/run-tests.yaml

```diff
@@ -87,15 +87,15 @@ jobs:
           sleep 10 # wait for initial servers to declare blocks, then let server decide which blocks to serve
 
           python -m petals.cli.run_server --converted_model_name_or_path $MODEL_NAME --block_indices 0:6 \
-            --initial_peers $INITIAL_PEERS --throughput 1 --torch_dtype float32 &> server3.log &
+            --initial_peers $INITIAL_PEERS --throughput 1 --tensor_parallel_devices cpu cpu --torch_dtype float32 &> server3.log &
```

Comment:
The intent is to make sure that there is at least one tensor-parallel server in every chain